modules/virt-about-virtualization-overview.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-vm-dashboard.adoc
+//
+
+[id="virt-about-reviewing-top-consumers_{context}"]
+= About reviewing top consumers
+
+In the {VirtProductName} dashboard, you can select a specific time period and view the top consumers of resources within that time period. Top consumers are virtual machines or `virt-launcher` pods that are consuming the highest amount of resources.
```

Comment:
s/are consuming/consume

Reply:
This module was replaced by a newer module that has been reviewed. Removing this module.

Reply:
This module does not belong to this PR. I had planned to use it in this PR, but decided not to. It belongs to the Dashboard assembly. When I deleted, Travis was not happy. So this module has been added back, but not being used in this assembly.